googlemock/test/gmock-matchers_test.cc

```diff
@@ -58,6 +58,11 @@
 # include <forward_list>  // NOLINT
 #endif
 
+// Disable MSVC2015 warning for std::pair: "decorated name length exceeded, name was truncated".
```

Comment:
I think these changes fix compiler warning:
https://github.com/google/googletest/pull/1218/commits/9c1bb542fb283e1d1f7f8ab1e3554b133d90160b

I'll check that

Reply:
VS 2015 produce this error:
```
     7>D:\Work\googletest\googlemock\test\gmock-matchers_test.cc(5683): error C2220: warning treated as error - no 'object' file generated [D:\Work\goo
       gletest\build\Win32\googlemock\gmock-matchers_test.vcxproj]
     7>D:\Work\googletest\googlemock\test\gmock-matchers_test.cc(5683): warning C4503: 'std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std
       ::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<He
       ad,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::p
       air<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,
       std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair
       <Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Head,std::pair<Matcher1,Matcher2>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
       >>>>>>>>>>>>>::pair': decorated name length exceeded, name was truncated [D:\Work\googletest\build\Win32\googlemock\gmock-matchers_test.vcxproj]
                 with
                 [
                     Head=testing::internal::NeMatcher<int>,
                     Matcher1=testing::internal::NeMatcher<int>,
                     Matcher2=testing::internal::NeMatcher<int>
                 ]
```

VS2017 compiles without warnings (maybe because of variadic templates)